Lib/multiprocessing/util.py

```diff
@@ -102,6 +102,29 @@ def log_to_stderr(level=None):
     _log_to_stderr = True
     return _logger
 
+
+# Abstract socket support
+
+def _platform_supports_abstract_sockets():
+    if sys.platform == "linux":
+        return True
+    if hasattr(sys, 'getandroidapilevel'):
+        return True
+    return False
```

Comment:
Would it be expensive to create an abstract UNIX socket and close it, to check if we get an error, rather than checking for the platform name? This approach is preferred in test.support since it's easy to implement and reliable. The difficulty is to generate an unique name for the test.

Reply:
I think that is a bit overkill, no? Specialy since such operation will be executed when the module is loaded and I dislike having side-effects.

Reply:
Ok, leave it as it is.